docs/tutorials/tut-setup-dev.md

```diff
@@ -114,52 +114,44 @@ We assume you have an Operating System and that is working :)
 
 #### Python and pyenv
 
-You will need both `python2` and `python3` installed on your system. While there are multiple ways to achieve this, we recommend using `pyenv`. At the time of writing, the latest versions of Python are *2.7.17* and *3.7.5*, so we're going to use these.
+You will need `python3` installed on your system. We recommend using `pyenv`. At the time of writing, the latest version of Python 3.7 is *3.7.5*.
```

Comment:
i think the latest python version is 3.8.2

Reply:
are we good with 3.8 instead of 3.7?

Reply:
I will keep 3.7 for now as I have a partner trying it today, will open another PR for 3.8 tomorrow